accounts/models.py

```diff
@@ -4,10 +4,11 @@
 
 
 class Registration(models.Model):
-    email = models.CharField(max_length=55, blank=True, default="")
+    email = models.CharField(max_length=50, blank=True, default="")
     phone = models.CharField(max_length=10, blank=True, default="")
```

Comment:
Phone number should be declared as IntegerField.

Reply:
will take care in next issue.